### DISCUSSION TITLE
hacked-SYSTEM.VSB appears cannot find file SYSTEM.VSB

### DISCUSSION THREAD

**Discussion by oracle000000** [bangcorrupt/hacktribe#240] [Q&A] [answered]
Hi
When I run the python script following instructions from you tube to change the header from hacked-SYSTEM.VSB to SYSYEM.VSB nothing appears in my folder but I have specified either synth/sampler and nothing appears. I am one step away from getting it done. Is this a windows thing?

**Comment by oracle000000** [comment]
i was able to fix problem

**Comment by bangcorrupt** [comment, marked as answer]
`e2-header.py` takes 2 arguments, the first is the path of the file to edit, the second is either `synth` or `sampler`.

**Reply by oracle000000** [reply]
Thankyou for the swift reply I was able to fix the issue😁
